src/GeositeFramework/region.json

```diff
@@ -8,7 +8,8 @@
         "url": "http://www.azavea.com/"
     },
     "headerLinks": [
-        { "text": "Menu", "url":"#", "items": [
+       { "text": "Getting started", "launchpad": true, "launchpadId": 1, "url": "javascript:;" },
```

Comment:
Looks like the cs template for creating the launchpad link inherently puts in an href of `javascript:;`, can this be omitted here?